vpr/src/route/check_rr_graph.cpp

```diff
@@ -124,16 +124,16 @@ void check_rr_graph(const t_graph_type graph_type,
             t_rr_type to_rr_type = device_ctx.rr_nodes[to_node].type();
 
             /* Only expect the following cases to have multiple edges
```

Comment:
Suggest a longer comment here.  Here's a proposal.
/* It is unusual to have more than one programmable switch (in the same direction) between a from_node and a to_node, as the duplicate switch doesn't add more routing flexibility. However, such duplicate switches can occur for some types of nodes, which we allow below. Reasons one could have duplicate switches between two nodes include:
 - The two switches have different electrical characteristics.
 - Wires near the edges of an FPGA are often cut off, and the stubs connected together. A regular switch pattern could then result in one physical wire connecting multiple times to other wires, IPINs or OPINs.
*/

Reply:
Done